tests/integration/oauth_adapter_test.py

```diff
@@ -54,7 +54,8 @@ def authorize_with_providers(cls, oauth_adapters):
             print("Please go to %s to authorize access: %s" % (provider, authz_url))
             print("Please copy/paste the \"code\" parameter from the resulting URL: ")
             sys.stdout.flush()
-            auth_code = sys.stdin.readline().strip()
+            # auth_code = sys.stdin.readline().strip()
+            auth_code = "X"
```

Comment:
?

Reply:
So, before we had the mock-provider, the tests would wait for me to click the authz link, get an access code, and then paste it in here.  This was handy for confirming that Bond worked with  the Fences.  With the mock-provider in place, the access code is irrelevant, the mock-provider always authorizes.  

I left the code commented out so that we could still uses if we want to test against a real provider for some reason.